connector_helpscout/models/helpscout_mailbox/importer.py

```diff
@@ -35,13 +33,28 @@ class HelpScoutMailboxImporter(Component):
     _apply_on = 'helpscout.mailbox'
 
     def _after_import(self, binding):
-        """Add HelpScout status choices as project stages"""
+        """
```

Comment:
Blank space is against [PEP-0257](https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings)